analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -143,6 +148,61 @@ int NullCoalescingAssignment(int? param)
     }
 }
 
+class AssignmentAndDeconstruction
+{
+    void TypeInference()
+    {
+        (int? discard, int? b) = (null, null);
+        _ = b.Value;                                         // FN: b is empty
+    }
+
+    void FirstLevel()
+    {
+        var (b, _) = (null as bool?, null as bool?);
+        _ = b.Value;                                         // FN: b is empty
+    }
+
+    void SecondLevel()
+    {
+        (int? b1, (int? b2, (int? b3, int? b4), int? b5)) = (42, (42, (42, null), null));
+        _ = b5.Value; // Compliant, FN
+    }
+
+    void ThirdLevel()
+    {
+        (int? b1, (int? b2, (int? b3, int? b4), int? b5)) = (42, (42, (42, null), null));
+        _ = b2.Value; // Compliant
```

Comment:
b1 is missing

Reply:
Added. 
The idea here was to test the 3rd level of nesting, but still capture everything into local variables, because there's a dedicated test for discard at multiple levels of nesting. 
I have now declared variables only at the level of nesting at which the test is testing value access.